.github/actions/pr-to-update-go/pr_to_update_go/__main__.py

```diff
@@ -0,0 +1,32 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import sys
+
+from github.MainClass import Github
+
+from pr_to_update_go.GoPRMaker import GoPRMaker
+from pr_to_update_go.constants import ENV_GITHUB_TOKEN
+
+
+def main() -> None:
+    try:
+        github_token: str = os.environ[ENV_GITHUB_TOKEN]
+    except KeyError:
+        print(f'Environment variable {ENV_GITHUB_TOKEN} must be defined.')
+        sys.exit(1)
+    gh = Github(login_or_token=github_token)
+    GoPRMaker(gh).run()
+
+
+main()
```

Comment:
I _think_ this should be wrapped in an `if __name__ == "__main__":` but I could be wrong; I'm not entirely sure the interpreter sets `__name__` like that when the module is called with `python -m`.

It's not normal to import modules beginning with `_` from packages - and nothing is meant to use this - so it could be safe, but it damages my delicate sensibilities to have large routines running as side effects to importing a module.

Reply:
If the filename is `__main__.py`, then `__name__` is always equal to `'__main__'`.

Reply:
Oh, right. That's an interesting edge case.